### DISCUSSION TITLE
There is no Forgot Password when I try to log in to the system

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16856] [How to] [answered]
Hello
One user forgot his password.
On the landing page there is no "forgot password" button in my YetiCRM system.
I checked this video and a button should be there:
https://www.youtube.com/watch?v=0k_ifsofCOc&list=PLX0lktP3Kukg_Da-TaEBys-aON68Yy2vh&t=8s&ab_channel=YetiForce

**This is what I have. I tried several browsers, etc.** 
Do I need to enable it somewhere?
![image](https://user-images.githubusercontent.com/20862325/216637927-f3806511-a2bc-4a92-b7ee-25c9fd59c944.png)

Also I checked gitstable  and there is no forgot password field also.
https://gitstable.yetiforce.com/index.php
![image](https://user-images.githubusercontent.com/20862325/216638188-ce8e2e0b-982f-4104-a431-bba52e2345bd.png)

**Comment by rskrzypczak** [comment, marked as answer]
Two conditions must be met for the button to be available:
1. Active option in the configuration file.
    https://github.com/YetiForceCompany/YetiForceCRM/blob/844a380a9292092ed648d3efe41c5fb2825d38a0/config/Security.php#L26-L27
2. SMTP configured (_Software configuration/Mail tools/SMTP configuration_)
     After all, access recovery is based on e-mail communication.
